supervision/detection/core.py

```diff
@@ -710,6 +710,57 @@ def from_paddledet(cls, paddledet_result) -> Detections:
             class_id=paddledet_result["bbox"][:, 0].astype(int),
         )
 
+    @classmethod
+    def from_depthai(cls, depthai_results) -> Detections:
```

Comment:
Could we rename it from `from_depthai` to `from_luxonis`? I think the Luxonis brand is much more recognizable than DepthAI. To be honest, I did not know what DepthAI is, but I'm absolutely familiar with Luxonis.